dotcom-rendering/src/components/MaintainAspectRatio.tsx

```diff
@@ -21,6 +21,25 @@ export const MaintainAspectRatio = ({ height, width, children }: Props) => (
 				top: 0;
 				left: 0;
 			}
+
+			/*
```

Comment:
Actually I'm wondering if it might be cleaner to move this styling to the YoutubeAtom container as it only applies there.

Reply:
Good shout